vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/KeyboardActions.java

```diff
@@ -75,4 +76,23 @@ public void typeText(WebElement element, String text)
             webDriverProvider.getUnwrapped(HidesKeyboard.class).hideKeyboard();
         }
     }
+
+    /**
+     * Clear the <b>element</b>'s text
+     * <br>
+     * The atomic actions performed are:
+     * <ol>
+     * <li>check if the element's text is empty</li>
+     * <li>if the element's text is not empty then clear the element, otherwise do nothing</li>
+     * </ol>
+     * @param element element to clear, must not be {@code null}
+     */
+    public void clearText(WebElement element)
+    {
+        String elementText = element.getText();
+        if (!StringUtils.isEmpty(elementText))
```

Comment:
why is this check needed?

Reply:
getText is ~5 times faster than clear command, so if field is empty we will not spend time on clearing it

Reply:
Android platform: ~300ms vs ~1600ms

Reply:
So in the most common case it is 1600 + 300 = 1800 ? :)